package.json

```diff
@@ -4,7 +4,8 @@
   "description": "API to process HTTP requests to access information on local cats",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "node ./bin/www",
```

Comment:
This also causes an error for me, I'm less sure of what the cause is, so might need some help from you debugging what this is trying to do:

> Error: Cannot find module '/Users/jamielynch/repos/cat-api/bin/www'